packages/common-integration-tests/src/vector-control-plane.ts

```diff
@@ -13,6 +13,7 @@ import {
   ListVectorIndexes,
   MomentoErrorCode,
 } from '@gomomento/sdk-core';
+import {VectorSimilarityMetric} from '@gomomento/sdk-core/dist/src/internal/clients';
```

Comment:
Should import from `'@gomomento/sdk-core'` after exporting from the top level of core.